javascript-data-model/order-history.js

```diff
@@ -0,0 +1,81 @@
+var orderFour = [
+  {
+    name: 'GameCube Controller'
+  },
+  {
+    name: 'The Art of Sql'
+  }
+];
+
+var orderHistory = [
+  {
+    name: 'Javascript for impatient programmers',
+    author: 'Dr. Axel Rauschmayer',
+    orderDate: 'August 4, 2020',
+    ifDelivered: true,
+    deliveryDate: 'August 8,2020',
+    subtotal: 31.55,
+    grandTotal: 34,
+    recipient: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    returnWindowOpen: false,
+    returnWindowDate: 'Sep 7, 2020',
+    reviewWritten: false
+  },
```

Comment:
In this object, data on the order is intermixed with data about the specific item ordered. This can be confusing and is unintuitive for immediately understanding the data. You should instead have data on the order itself in this object, then in this object, have a property that stores data on the item(s) ordered.